.github/workflows/slow-checks.yml

```diff
@@ -45,3 +45,39 @@ jobs:
           name: coverage-python${{ matrix.python-version }}-slow
           path: .coverage
           include-hidden-files: true
+
+  runner-images:
```

Comment:
You could also define a job with a yaml matrix that calls a workflow call that runs the test. This way, you don't need to mess with bash.